app/assets/javascripts/application/filter-list-items.js

```diff
@@ -40,7 +40,7 @@
       filter._trackTimeout = root.setTimeout(function(){
         var pagePath = window.location.pathname.split('/').pop();
         if(pagePath){
-          window._gaq && _gaq.push(['_trackEvent', 'searchBoxFilter', search, pagePath, 0, true]);
+          GOVUK.analytics.trackEvent('searchBoxFilter', search, {label: pagePath, value: 0, nonInteraction: true});
```

Comment:
The `value: 0` part can be omitted now. The 0 was only added before so that the final argument could be included. This is the default behaviour of the classic tracker.